packages/limit-service/lib/config.js

```diff
@@ -27,5 +27,7 @@ module.exports = {
             return result.count;
         }
     },
-    customThemes: {}
+    customThemes: {
```

Comment:
**Nothing to change here**. Wanted to document a thought as it came to my mind and then reached the same naming conclusion after thinking through it.  The idea was renaming this config property to something more generic like `allowedThemes`. I scraped this idea in my head because (1) this configuration name has already been used and probably was given some thought and (2) it's probably best to stay with this name as it is not just "allowing" for themes but also identifies that this check is like a flag that can be disabled